docs/index.html

```diff
@@ -34,7 +34,7 @@
 If you have any questions, take a look at the <a href="https://github.com/Leaflet/Leaflet/blob/master/FAQ.md">FAQ</a> first.</p>
 
 
-</div>
+</main>
```

Comment:
So the problem is that `class="container"` now wraps "Trusted by" per https://github.com/Leaflet/Leaflet/pull/7901/.

`<main>` should only be used once per document though, so another solution could be this structure:

```html
<!-- docs/_layouts/v2.html -->
<main>
  <div class="container">
    {{ content }}
  </div>
</main>
```

and in docs/index.html change these `<main>` back to `<div>` for `class="container"`. Does that make sense @Falke-Design?

Reply:
Thx changed